portfolio/src/main/webapp/script.js

```diff
@@ -170,6 +170,13 @@ function createCommentElement(comment) {
   const commentText = document.createElement('div');
   commentText.innerText = comment.text;
 
+  // Set the commentText's color property corresponding to the sentimentScore
+  const commentTextColor = getSentimentColor(comment.sentimentScore);
+  commentText.style.color = 'rgb(' + commentTextColor.r + 
+                            ',' + commentTextColor.g + 
+                            ',' + commentTextColor.b +
+                            ')'; 
```

Comment:
You could also write
`` `rgb(${ commentTextColor.r },${ commentTextColor.g },${ commentTextColor.b })` ``